pkg/module/parse/template_to_module.go

```diff
@@ -241,3 +263,54 @@ func insertLayerIntoManifest(
 
 	return nil
 }
+
+func (p *Parser) fetchAssociatedResourcesField(ctx context.Context, layer *img.OCI, version string) ([]string,
+	error,
+) {
+	associatedResourcesLayer := v1beta2.ImageSpec{
```

Comment:
Creating this seems a bit redundant since the only value used for a dependency is `layer.CredSecretSelector` for the `LookupKeyChain()`